features/site.feature

```diff
@@ -31,6 +31,17 @@ Feature: Site Command
     site.test
     """
 
+  Scenario: Add alias domain
+    When I run 'bin/ee site update site.test --add-alias-domains=alias.site.test '
+    And I run '/bin/bash -c 'echo "127.0.0.1 alias.site.test" >> /etc/hosts''
```

Comment:
I think we should be doing this in EE itself. Can you take this up?

Reply:
@kirtangajjar I cannot take this up as of now. Please create an issue we will see it later in next release.

Reply:
Ohh. It's already there - https://github.com/EasyEngine/easyengine/issues/1460